ext/rbs_extension/location.c

```diff
@@ -224,10 +227,16 @@ static VALUE rbs_new_location_from_loc_range(VALUE buffer, rbs_loc_range rg) {
   return obj;
 }
 
+static rbs_constant_id_t rbs_constant_pool_find_ruby_symbol(VALUE symbol) {
+  VALUE name = rb_sym2str(symbol);
+
+  return rbs_constant_pool_find(RBS_GLOBAL_CONSTANT_POOL, (const uint8_t *) RSTRING_PTR(name), RSTRING_LEN(name));
+}
+
 static VALUE location_aref(VALUE self, VALUE name) {
   rbs_loc *loc = rbs_check_location(self);
 
-  rbs_constant_id_t id = rbs_find_constant_id_from_ruby_symbol(name);
+  rbs_constant_id_t id = rbs_constant_pool_find_ruby_symbol(name);
```

Comment:
This is just a rename, but keeps the `find` semantics. So doing `loc[name]` with a bunch of names will _not_ permenantly leak a bunch of constants into the global pool.